Add Axe for automated accessibility feedback, address issues highlighted by Axe

Noah-T opened the pull request:
This PR does three things:
1. Add [Axe](https://www.npmjs.com/package/@axe-core/react) for automating (some) accessibility checks.
2. Replace the outer div in the page with a `<main>` tag (recommendation from Axe)
3. Slightly darken buttons to improve contrast with white text (recommendation from Axe). Axe was most cranky with the lack of contrast between the lighter red button and the white text, so I made the buttons darker. I then made the banner a dark blue, because I thought it looked good with the dark red. We could always just change the button color if people don't like the blue banner. 

This is the color scheme present in the UI right now:
<img width="506" alt="Screen Shot 2021-02-15 at 3 38 52 PM" src="https://user-images.githubusercontent.com/6364852/107991112-f1157600-6fa3-11eb-8cb6-1ee25752a6af.png">

This is the color scheme that would apply if this PR is merged:
<img width="454" alt="Screen Shot 2021-02-15 at 3 40 11 PM" src="https://user-images.githubusercontent.com/6364852/107991235-2b7f1300-6fa4-11eb-8039-1605f4ddbd75.png">

Noah-T commented:
@MattRC7, I hear you have some experience with accessibility best practices. Feel free to let me know if you have any suggestions. Thanks!

Noah-T commented:
@livgust this is ready for review whenever you have some time.

harcod commented:
I tested this PR and it worked for me.  Looks better IMO. The new color is not so "stop sign red".

MattRC7 commented:
Hoping to take a look some time tonight.

MattRC7 commented on package.json:
```diff
@@ -1,42 +1,45 @@
-            "last 1 safari version"
-        ]
-    }
+  "name": "ma_covid_vaccine_website",
```
Can you make sure you're using recently merged prettier config? I believe, unless if changed, that we are using 4 space indents. You can run prettier on the whole workspace if you wish.

MattRC7 commented on src/App.js:
```diff
@@ -3,50 +3,53 @@ import React from "react";
-                            Adams. All rights reserved.
-                        </Typography>
+        <MuiThemeProvider theme={theme}>
+            <>
```
If we're using the theme provider, we can remove this fragment.

MattRC7 commented on src/App.js:
```diff
@@ -3,50 +3,53 @@ import React from "react";
+        <MuiThemeProvider theme={theme}>
+            <>
+                <Menu />
+                <div className={classes.main} role="main">
```
Can we convert this div to a `main` tag instead of using the role. In general, semantic HTML is preferred over explicit aria roles, where feasible.

MattRC7 reviewed (commented):
Left a few comments via mobile. Will try to tale a look on desktop later. Thanks for opening this!

MattRC7 commented:
Additionally, if you could add a section to the README.md explaining how to use the aXe tool, that would be helpful. Looks like you need to open a browser dev console window in order to see the results?

Noah-T commented:
I think I addressed all comments here.

orzechowskid commented:
looks like your latest commit accidentally removed the `role` attribute from the primary container `<div>` in App.js .  https://github.com/Noah-T/macovidvaccines.com/commit/fbb1bf87cb64753ad836de3008505ddd4c5ec796#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67

(though I agree with @MattRC7 that the HTML5 `<main>` tag is a better fit than a `<div>` here)

Noah-T commented:
> looks like your latest commit accidentally removed the `role` attribute from the primary container `<div>` in App.js . [Noah-T@fbb1bf8#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67](https://github.com/Noah-T/macovidvaccines.com/commit/fbb1bf87cb64753ad836de3008505ddd4c5ec796#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67)
> 
> (though I agree with @MattRC7 that the HTML5 `<main>` tag is a better fit than a `<div>` here)

@orzechowskid, I intentionally removed the role attribute based on this comment: https://github.com/livgust/macovidvaccines.com/pull/18#discussion_r576453314. The `<main>` tag has the same effect, so `role=main` is no longer necessary.

orzechowskid commented:
got it.  I didn't see a `<main>` tag in the PR either but I'm probably just missing something.

(also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

Noah-T commented:
> got it. I didn't see a `<main>` tag in the PR either but I'm probably just missing something.
> 
> (also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

I had dropped `main` by accident--probably a bad merge on my part. Thanks for catching that. And glad to hear you had a positive experience with axe.

livgust reviewed (approved):
This looks great to me! @MattRC7 do you have a moment today to give a second sign-off? If you do, I'll merge once you approve (or you can if it lets you)

MattRC7 commented:
I can approve after work this evening. I do not have merge power though.

MattRC7 reviewed (approved):
Great work!

bernfarr commented:
> (also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

Should running `axe` be added as part of the getting-started directions?
